03.bowling/bowling.rb

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+class BowlingScoreCalculator
+  def initialize(scores_str)
+    @scores_str = scores_str
+  end
+
+  def print_total_score
+    scores_by_bowl = split_scores_by_bowl(scores_str: @scores_str)
+    scores_by_frame = group_scores_by_frame(scores_by_bowl:)
+    puts calculate_total_score(scores_by_frame:)
+  end
+
+  private
+
+  def split_scores_by_bowl(scores_str:)
```

Comment:
```suggestion
  def split_scores_by_bowl(scores_str)
```

キーワード引数にするメリットがあまりないので、普通の引数にしましょう。
group_scores_by_frame(scores_by_bowl:) も同様です